kafka_utils/util/metadata.py

```diff
@@ -15,7 +15,7 @@
 from kafka import KafkaClient
 
 
-def _prepare_host_list(broker_list):
+def prepare_host_list(broker_list):
```

Comment:
I don't think this function is being used anywhere in the code. Can we remove it?